src/ray/object_manager/ownership_based_object_directory.cc

```diff
@@ -80,11 +80,19 @@ ray::Status OwnershipBasedObjectDirectory::ReportObjectAdded(
   request.set_node_id(node_id.Binary());
 
   rpc_client->AddObjectLocationOwner(
-      request, [worker_id, object_id](Status status,
-                                      const rpc::AddObjectLocationOwnerReply &reply) {
+      request, [worker_id, object_id, node_id](
+                   Status status, const rpc::AddObjectLocationOwnerReply &reply) {
         if (!status.ok()) {
-          RAY_LOG(ERROR) << "Worker " << worker_id << " failed to add the location for "
-                         << object_id;
+          if (status.IsObjectNotFound()) {
+            RAY_LOG(INFO) << "Worker " << worker_id << " failed to add the location "
+                          << node_id << " for " << object_id
+                          << " because the owner no longer has the object; we assume the "
+                             "object was evicted.";
+          } else {
+            RAY_LOG(WARNING) << "Worker " << worker_id << " failed to add the location "
```

Comment:
This can happen when the owner is dead right? We should probably make it INFO to avoid spam msgs?

Reply:
Yep, when the owner is dead, and thanks I wasn't sure if this should be INFO or WARNING, I'll change it to INFO.